spec/features/orders/user_orders_index_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe "A user sees thier completed orderss" do
+  scenario "logged-in user can see all completed orders" do
+
+    user = create(:user)
+    user_order1 = create(:order, user_id: user.id)
+    user_order2 = create(:order, user_id: user.id)
+
```

Comment:
I think we might be able to make this a bit more robust by stubbing out the current_user, and then using the current_user method in the order's controller action instead of finding by the id passed in.